tasks/run_cd4pe_job.rb

```diff
@@ -53,6 +53,10 @@ def self.preserve_symlink(entry, dest)
     Gem::Package::TarReader.new( Zlib::GzipReader.open zipped_file_path ) do |tar|
       dest = nil
       tar.each do |entry|
+        # skip 'PaxHeaders.X' entries as they interfere with the TAR_LONGLINK logic
+        if !(entry.directory? || entry.file? || entry.header.typeflag == SYMLINK_SYMBOL) && entry.full_name != TAR_LONGLINK
```

Comment:
I noticed when I printed the `entry.header.typeflag` for a PaxHeader it was always "x". Perhaps we can change this check to explicitly check for this value?

We can add a PAX_HEADER variable alongside TAR_LONGLINK and SYMLINK_SYMBOL above like this

```ruby
        PAX_HEADER = 'x'.freeze
```
Then change the if statement to 

```ruby

        if entry.header.typeflag == PAX_HEADER
          next
        end
```

Reply:
Yeah thats def much nicer